tools/logicalimager/tsk_logical_imager.cpp

```diff
@@ -672,7 +672,7 @@ static BOOL getPhysicalDrives(std::vector<std::wstring> &phyiscalDrives) {
             }
         }
     } else {
-        consoleOutput(stderr, "QueryDosDevice() return error: %d\n", GetLastError());
+        fprintf(stderr, "QueryDosDevice() return error: %d\n", GetLastError());
```

Comment:
Change all these back to consoleOutput?

Reply:
When these functions are called, the output directory and the console.txt file are not created yet.

Reply:
If you add a null check to logOutputToFile, you could always call consoleOutput instead of having to keep track of whether the code executes before or after the log file is created.

Reply:
Change this and the other fprints back to consoleOutput now that it does the null check?